literate-calc-mode.el

```diff
@@ -41,6 +42,23 @@
 ;; TODO org-babel-execute
 ;; TODO org export
 
+(defgroup literate-calc-mode nil
+  "Display inline results from calc."
+  :group 'editing
+  :prefix "literate-calc-mode-")
+
+(defcustom literate-calc-mode-inhibit-line-functions '(literate-calc-mode-inhibit-in-src-blocks)
+  "Hook functions called for each line to test whether to inhibit calculation.
+
+If any of these functions returns non-nil, overlays will not be displayed."
+  :group 'literate-calc-mode
+  :type 'hook)
+
+(defun literate-calc-mode-inhibit-in-src-blocks ()
+  "Return non-nil if point is in a source block."
+  (memq (org-element-type (org-element-context))
+        '(inline-src-block src-block)))
```

Comment:
Thinking about it, we probably want to test that we're in an org buffer before calling this. I didn't think about it before cos I only enable this mode in org buffers. :)

```suggestion
  (when (derived-mode-p 'org-mode)
    (memq (org-element-type (org-element-context))
          '(inline-src-block src-block))))
```